adafruit_motor/servo.py

```diff
@@ -71,6 +71,25 @@ def fraction(self, value):
         duty_cycle = self._min_duty + int(value * self._duty_range)
         self._pwm_out.duty_cycle = duty_cycle
 
+    @property
+    def pulse_width(self):
+        """The pulse width sent to the servo in microseconds. Must be in
+        the range ``min_pulse`` to ``max_pulse``. Is None when servo is disabled.
+        """
+        if self._pwm_out.duty_cycle == 0:  # Special case for disabled servos
+            return None
+        return self._pwm_out.duty_cycle
```

Comment:
This is incorrect because it's not doing the reverse math of line 88.

Reply:
Whoops, thanks for the catch!